Arriba/Arriba.Web/configuration.default/Configuration.jsx

```diff
@@ -48,7 +48,7 @@
             overview:(
                 <span>
                     Arriba is a data exploration, free text search, and structured search engine.
-                    Need more <a href="Search.html?help=true">help</a>?
+                    Need more <a href="/?help=true">help</a>?
```

Comment:
There is a bug. I actually agree with it bug, but it just hasn't been urgent. I'll just get it out of the way then.